src/apps/ENGINE/src/Main.cpp

```diff
@@ -58,7 +58,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PSTR szCmdLine,
         return EXIT_SUCCESS;
     }
 
-    SDL_InitSubSystem(SDL_INIT_EVENTS | SDL_INIT_VIDEO | SDL_INIT_GAMECONTROLLER);
```

Comment:
Event subsystem is initialized by default, right?

Reply:
Right, it is implied by SDL_INIT_VIDEO